src/extensions/reactNative.ts

```diff
@@ -73,7 +68,11 @@ function attach (plugin, command, context) {
     log('initializing react native')
     log(cmd)
     const withTemplate = reactNativeTemplate ? ` with ${print.colors.cyan(reactNativeTemplate)} template` : ''
-    const spinner = print.spin(`adding ${print.colors.cyan('React Native ' + reactNativeVersion)}${withTemplate} ${print.colors.muted('(30 seconds-ish)')}`)
+    const spinner = print.spin(
+      `adding ${print.colors.cyan('React Native ' + reactNativeVersion)}${withTemplate} ${print.colors.muted(
+        '(30 seconds-ish)',
```

Comment:
We should probably change this to `1-2 minutes-ish`

Reply:
I've been timing it and it regularly hits about 30 seconds on my machine (which isn't a fast one).

Reply:
Regarding the `git move` comment above, I tried that 3 different ways, and git refused to recognize it. 🤷‍♂️  I even blew away all my commits and did them one at a time, but it didn't work.